### PR TITLE
RPET-661: add location to bulk scheduler dropdown

### PR DESCRIPTION
**Pull request by bendoc1996**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RPET-661

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: definitions/divorce/json/FixedLists/FixedLists.json [hunk 1]
```diff
@@ -561,7 +561,8 @@
     "LiveFrom": "01/01/2017",
     "ID": "courtEnum",
     "ListElementCode": "buryStEdmunds",
-    "ListElement": "Bury St Edmunds"
+    "ListElement": "Bury St Edmunds",
+    "DisplayOrder": 3
```
Comment (Cotterale) [inline definitions/divorce/json/FixedLists/FixedLists.json:565]: You've added display order 3 here for the ID "courtEnum" and then display orders 1-2 and 4-10 for the ID "DnCourtEnum".

File: definitions/bulk-action/json/FixedLists.json [hunk 2]
```diff
@@ -88,5 +97,12 @@
     "ID": "HasJudgePronouncedEnum",
     "ListElementCode": "Yes",
     "ListElement": "Yes"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "DnCourtEnum",
+    "ListElementCode": "buryStEdmunds",
+    "ListElement": "Bury St Edmunds",
```
Comment (Cotterale) [inline definitions/bulk-action/json/FixedLists.json:105]: Just a small thing, could this be moved up to match the display order?

**Review by Cotterale — APPROVED** [review]

File: definitions/divorce/json/FixedLists/FixedLists.json (hunk 1 above)
Comment (Cotterale) [reply definitions/divorce/json/FixedLists/FixedLists.json:565]: Just curious if this was meant to be done for "DnCourtEnum"

**Review by HarryH96 — APPROVED** [review]

**Review by TSeelig — APPROVED** [review]

**Comment by bendoc1996** [comment]
close bad PR

**Pull request closed without merging**